main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+// Add the database path
+const DB string = "./data.csv"
+
+// Create pokemon struct
+type Pokemon struct {
+	Id int `json:"id"`
+	Name string `json:"name"`
+}
+
+// Global array for pokemons
+var Pokemons []Pokemon
+
+// readDB reads a CSV file and return the matrix of entries
+func readDB() [][]string{
+	log.Println("INFO - Reading DB")
+
+	var file, err = os.OpenFile(DB, os.O_RDWR, 0644)
+	if err != nil {
+		log.Fatalf("ERROR - %s", err.Error())
+	}
+	defer file.Close()
```

Comment:
file.Close() can return an error if something fails, you can try something like this to manage it:
```suggestion
defer func(){
  err := file.Close()
  if err != nil {
    log.Errorf("file cannot be closed: %v", err)
  }
}
```